plugins/azure-devops-backend/config.d.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export interface Config {
+  /** Configuration options for the azure-devops-backend plugin */
+  azureDevOps?: {
+    /**
+     * The hostname of the given Azure instance
+     */
+    host: string;
+    /**
+     * Token used to authenticate requests.
+     * @visibility secret
+     */
+    token?: string;
+    /**
+     * The organization of the given Azure instance
+     */
+    organization: string;
+    /**
+     * The max number of items to return - applies to builds
+     */
+    top: number;
```

Comment:
I guess this one feels a little unexpected as a config parameter - feels like it should be frontend driven or so?

Reply:
I've moved this so it will be coming from the future frontend but now I feel like it's more confusing. I agree with your comment just unsure of my design to handle it now